### DISCUSSION TITLE
Abort current wire:navigate request

### DISCUSSION THREAD

**Discussion by pronode** [livewire/livewire#7476] [1. Help] [answered]
I'm working on extension (https://github.com/pronode/navigate-turbo) to speed up UI.
However, Livewire is not aborting previous wire:navigate action if user clicks other link meanwhile.

I would like to programatically call something like Livewire.stopNavigate() to abort current request when user clicks another link.

Is it possible?

**Comment by joshhanley** [comment, marked as answer]
We're in the process of investigating all wire:navigate issues and I have tested out your scenario. I believe this has been fixed, see https://livewire.laravel.com/docs/navigate#javascript-hooks for how to use `event.preventDefault()` to cancel, so I'm going to close this. If you're still experiencing problems, please open a new discussion. Thanks!
